daemon/api_debug.go

```diff
@@ -304,6 +304,7 @@ func getGadgetDiskMapping(st *state.State) Response {
 		switch sealingMethod {
 		case device.SealingMethodLegacyTPM, device.SealingMethodTPM:
 			encType = secboot.EncryptionTypeLUKS
+			// TODO:ICE: device setup hook support goes away
 		case device.SealingMethodFDESetupHook:
 			encType = secboot.EncryptionTypeDeviceSetupHook
```

Comment:
this code seems already a bit broken anyway because this sealing method shouldn't imply ICE?? so probably the TODO should become an XXX about double checking the intention here as well?

Reply:
I added more XXX here now, I hope in a followup I can remove the entire code.